src/stacks-hierarchy/Root_ParkingViolationsReporting/Root_ParkingViolationsSelect.tsx

```diff
@@ -0,0 +1,140 @@
+import {useState} from 'react';
+import {Linking} from 'react-native';
+import {SvgXml} from 'react-native-svg';
+import {ScrollView} from 'react-native-gesture-handler';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {Button} from '@atb/components/button';
+import {MessageBox} from '@atb/components/message-box';
+import {ThemeText} from '@atb/components/text';
+import {useTranslation} from '@atb/translations';
+import {ParkingViolationType} from '@atb/api/types/mobility';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy';
+import {ParkingViolationTexts} from '@atb/translations/screens/ParkingViolations';
+import {ScreenContainer} from './components/ScreenContainer';
+import {SelectGroup} from './components/SelectGroup';
+import {useParkingViolations} from './use-parking-violations';
+import {PermissionRequiredError} from './use-user-location';
+
+export type SelectViolationScreenProps =
+  RootStackScreenProps<'Root_ParkingViolationsSelect'>;
+
+const ICON_SIZE = 50;
+
+export const Root_ParkingViolationsSelect = ({
+  navigation,
+}: SelectViolationScreenProps) => {
+  const style = useStyles();
+  const {t} = useTranslation();
+  const {theme} = useTheme();
+  const {isLoading, isError, errors, violations} = useParkingViolations();
+  const [selectedViolations, setSelectedViolations] = useState<
+    ParkingViolationType[]
+  >([]);
+
+  return (
+    <ScreenContainer
+      leftHeaderButton={{type: 'close'}}
+      title={t(ParkingViolationTexts.selectViolation.title)}
+      buttons={
+        <Button
+          disabled={isError}
+          onPress={() => {
+            navigation.navigate('Root_ParkingViolationsPhoto', {
+              selectedViolations,
+            });
+          }}
+          text={t(ParkingViolationTexts.selectViolation.nextButton)}
+        />
+      }
+      isLoading={isLoading}
+    >
+      {isError && errors.map((error) => <ErrorMessage error={error} />)}
```

Comment:
It should be something to differ them, a message or something? If not we probably don't want to show both, and we can filter uniques before rendering?

Reply:
Solved it by only showing one of the errors. If both errors are present I assume that the location error is most relevant, since it can be the reason why loading fails.